primer3/argdefaults.py

```diff
@@ -42,5 +53,168 @@ class Primer3PyArguments:
     temp_only: int = 0
     calc_type_wrapper = 'ANY'
 
+    dmso_conc: float = 0.0
+    dmso_fact: float = 0.6
+    formamide_conc: float = 0.8
+    annealing_temp_c: float = 50.0  # 70.0 or 120 in primer3 source
+
     def todict(self) -> dict:
         return dataclasses.asdict(self)
+
+
+# ~~~~~~~ RUDIMENTARY PRIMER3 MAIN WRAPPER (see Primer3 docs for args) ~~~~~~ #
+def format_boulder_io(p3_args: Dict[str, Any]) -> bytes:
+    '''Convert argument dictionary to boulder formatted bytes
+
+    Args:
+        p3_args: primer3 arguments to format boulder style
+
+    Returns:
+        Boulder formatted byte string
+    '''
+    class TransformT:
+        comma_pair_list_space_separated = 'comma_pair_list_space_separated'
+        dashed_range = 'dashed_range'
+        path_fixer = 'path_fixer'
+        semicolon = 'semicolon'
+        space_separated = 'space_separated'
+        tuple_list_comma_separated = 'tuple_list_comma_separated'
+        debug = 'debug'
+
+    transform_map = {
+        'PRIMER_PRODUCT_SIZE_RANGE': TransformT.dashed_range,
+        'PRIMER_THERMODYNAMIC_PARAMETERS_PATH': TransformT.path_fixer,
+        'SEQUENCE_EXCLUDED_REGION': TransformT.comma_pair_list_space_separated,
```

Comment:
Additional `comma_pari_list_space_separated` target: `SEQUENCE_TARGET`

Reply:
removed this in the refactor to `wrap` and `unwrap` version of this code